### PR TITLE
Updated the `construct_runtime` from frame-v1 to frame-v3 style

### PR DESCRIPTION
**Pull request by abhi3700**
Changed these (& similar lines) all over the codebase as per FRAME v3:

```diff
- System: frame_system::{Pallet, Call, Config, Storage, Event<T>}
+ System: frame_system
```

### CODE REVIEW

File: frame/authorship/src/lib.rs [hunk 1]
```diff
@@ -80,7 +80,7 @@ impl<T: Config> Pallet<T> {
 	pub fn author() -> Option<T::AccountId> {
 		// Check the memorized storage value.
 		if let Some(author) = <Author<T>>::get() {
-			return Some(author)
+			return Some(author);
```
Comment (bkchr) [inline frame/authorship/src/lib.rs:83]:
```suggestion
			return Some(author)
```

File: frame/authorship/src/lib.rs [hunk 2]
```diff
@@ -165,7 +165,7 @@ mod tests {
 		{
 			for (id, mut data) in digests {
 				if id == TEST_ID {
-					return u64::decode(&mut data).ok()
+					return u64::decode(&mut data).ok();
```
Comment (bkchr) [inline frame/authorship/src/lib.rs:168]:
```suggestion
					return u64::decode(&mut data).ok()
```

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-check-benches
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/3008264

File: frame/benchmarking/src/tests.rs [hunk 3]
```diff
@@ -61,7 +61,7 @@ mod pallet_test {
 		}
 
 		pub fn always_error(_origin: OriginFor<T>) -> DispatchResult {
-			return Err("I always fail".into())
+			return Err("I always fail".into());
```
Comment (bkchr) [inline frame/benchmarking/src/tests.rs:64]:
```suggestion
			Err("I always fail".into())
```

**Review by bkchr — COMMENTED** [review]
Please revert these irrelevant changes (there are more in your pr)

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-check-benches
**Logs**: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/3008734

**Comment by abhi3700** [comment]
will create a separate PR later...
ARCHIVED.

**Pull request closed without merging**